src/db/index.ts

```diff
@@ -0,0 +1,31 @@
+import { MongoClient, ServerApiVersion } from 'mongodb';
+
+export async function createMongoClient(uri: string) {
+  return new MongoClient(uri, {
+    serverApi: {
+      version: ServerApiVersion.v1,
+      strict: true,
+      deprecationErrors: true,
+    },
+  });
+}
+// Create a MongoClient with a MongoClientOptions object to set the Stable API version
+export async function connectMongoDB(client: MongoClient) {
+  try {
+    // Connect the client to the server	(optional starting in v4.7)
+    await client.connect();
+    console.log('Connected');
```

Comment:
For the next PR we should replace all meaningful `console.log`-s with calls to the `fastify.logger`